README.md

```diff
@@ -7,45 +7,681 @@
+> 	 Executing job 1006707f-efa8-48c2-98af-a1b320a59780...
+> 	 Ensuring Hot-Storage satisfies the configuration...
+> 	 No actions needed in Hot Storage.
+> 	 Hot-Storage execution ran successfully.
+> 	 Ensuring Cold-Storage satisfies the configuration...
+> 	 Current replication factor is lower than desired, making 10 new deals...
+> 	 Calculating piece size...
+> 	 Estimated piece size is 256 bytes.
+> 	 Proposing deal to miner t01459 with 0 fil per epoch...
+> 	 Proposing deal to miner t0117734 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0120993 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0120642 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0121477 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0119390 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0101180 with 10000000 fil per epoch...
+> 	 Proposing deal to miner t0117803 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0121852 with 500000000 fil per epoch...
+> 	 Proposing deal to miner t0119822 with 500000000 fil per epoch...
+> 	 Watching deals unfold...
+> 	 Deal with miner t0117803 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121852 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121477 changed state to StorageDealClientFunding
+> 	 Deal with miner t0101180 changed state to StorageDealClientFunding
+> 	 Deal with miner t0119822 changed state to StorageDealClientFunding
+> 	 Deal with miner t0119390 changed state to StorageDealClientFunding
+> 	 Deal with miner t0120642 changed state to StorageDealClientFunding
+> 	 Deal with miner t0117734 changed state to StorageDealClientFunding
+> 	 Deal with miner t01459 changed state to StorageDealClientFunding
+> 	 Deal with miner t0120993 changed state to StorageDealClientFunding
+> 	 Deal with miner t0121477 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0119822 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0117734 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0121852 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t01459 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0120642 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0120993 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0117803 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0101180 changed state to StorageDealWaitingForDataRequest
+> 	 Deal with miner t0119390 changed state to StorageDealWaitingForDataRequest
+>    Deal with miner t01459 changed state to StorageDealProposalAccepted
+> 	 Deal with miner t01459 changed state to StorageDealSealing
+```
+
+The output will looks something like the above. With a little luck, you will start seeing some successful storage deals.
+
+To-do: Add section on recovering a bucket from an archive using the Lotus client.
```

Comment:
N: In the meantime, it might worth to paste a link to the blog post where there was a video about this?

Reply:
Sure, good idea